src/components/content/heroes/aurora/weather-easter-egg/clouds.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const MAX_CLOUDS_PER_GROUP = 5;
+
+interface CloudsProps {
+  amount: number;
+  type: CloudType;
+}
+
+interface CloudGroupProps {
+  type: CloudType;
+}
+
+export enum CloudType {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+const getRandomValue = (min: number, max: number) =>
+  Math.random() * (max - min) + min;
+const getRandomTransform = (distance: number) =>
+  `translate(${getRandomValue(-distance, distance)}px, ${getRandomValue(
+    -distance,
+    distance,
+  )}px)`;
+
+const cloudAnimation = (distance: number) =>
+  keyframes`
+  0% {
+    transform: ${getRandomTransform(distance)};
+  }
+  50% {
+    transform: ${getRandomTransform(distance)};
+  }
+  100% {
+    transform: ${getRandomTransform(distance)};
+  }`;
+
+const CloudsContainer = styled.div<{ type: CloudType }>`
+  width: 100%;
+  height: 100%;
+
+  background: linear-gradient(
+    180deg,
+    ${(props) => (props.type === CloudType.LIGHT ? '#8dedfd' : '#4a495d')} 0%,
+    ${(props) => (props.type === CloudType.LIGHT ? '#49d6f6' : '#8686ab')} 100%
+  );
+`;
+
+const CloudGroupContainer = styled.div`
+  width: 25%;
+  height: 200px;
+  position: absolute;
+
+  top: calc(${() => Math.random() * 100}% - 200px);
+  left: calc(${() => Math.random() * 100}% - 200px);
+
+  animation: ${() => cloudAnimation(50)} ${getRandomValue(9, 12)}s alternate
+    infinite;
+`;
+
+export const Clouds = ({ amount, type }: CloudsProps) => {
```

Comment:
cool and flexible component ✨ the cloudy one really looks depressingly cloudy 😅
<img width="1234" alt="image" src="https://github.com/satellytes/satellytes.com/assets/1701755/3e5fda83-9500-4bd0-8352-a846e063771b">

slightly cloudy is bad for readability (while all other effects work) and it doesn't look very convincing.
<img width="1249" alt="image" src="https://github.com/satellytes/satellytes.com/assets/1701755/d82aa22a-1b73-471e-a34e-521e6dad2aef">